### PR TITLE
Add elliptic HasConverged trigger

### PR DESCRIPTION
**Pull request by nilsvu**
## Proposed changes

Trigger when the solver has converged. Useful to output norms and volume data only of the solution.

### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `new feature` if appropriate.

### CODE REVIEW

**Comment by kidder** [comment]
marking as in progress until it compiles

**Review by wthrowe — DISMISSED** [review]

File: tests/Unit/Elliptic/Triggers/CMakeLists.txt [hunk 1]
```diff
@@ -5,11 +5,23 @@ set(LIBRARY "Test_EllipticTriggers")
...
+  )
+
+target_link_libraries(
+  ${LIBRARY}
+  PRIVATE
+  Convergence
+  DataStructures
+  Elliptic
+  Parallel
+  EventsAndTriggers
```
Comment (kidder) [inline tests/Unit/Elliptic/Triggers/CMakeLists.txt:24]: I think we can alphabetize these

File: src/Elliptic/Triggers/HasConverged.hpp [hunk 2]
```diff
@@ -0,0 +1,45 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
```
Comment (kidder) [inline src/Elliptic/Triggers/HasConverged.hpp]: unused

File: tests/Unit/Elliptic/Triggers/Test_HasConverged.cpp [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+#include "Framework/TestingFramework.hpp"
+
+#include <cstddef>
+#include <initializer_list>
+#include <memory>
+#include <pup.h>
+#include <string>
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "DataStructures/DataBox/Tag.hpp"
```
Comment (kidder) [inline tests/Unit/Elliptic/Triggers/Test_HasConverged.cpp]: unused

File: tests/Unit/Elliptic/Triggers/Test_HasConverged.cpp [hunk 4]
```diff
@@ -0,0 +1,60 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "Framework/TestingFramework.hpp"
+
+#include <cstddef>
+#include <initializer_list>
```
Comment (kidder) [inline tests/Unit/Elliptic/Triggers/Test_HasConverged.cpp]: unused

File: tests/Unit/Elliptic/Triggers/Test_HasConverged.cpp [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+#include <memory>
+#include <pup.h>
+#include <string>
```
Comment (kidder) [inline tests/Unit/Elliptic/Triggers/Test_HasConverged.cpp]: unused

**Review by kidder — CHANGES_REQUESTED** [review]
you can squash

**Comment by nilsvu** [comment]
Fixed and rebased, thanks for reviewing @kidder and @wthrowe

**Review by kidder — APPROVED** [review]

**Review by wthrowe — APPROVED** [review]

**Pull request merged**